blocks/header/nav-toolbar.js

```diff
@@ -0,0 +1,46 @@
+import decorateSearch from "../search/search.js";
```

Comment:
this file has the most significant refactor; let me know if you'd like to talk through it